conf/vertebrates/allowed_species.json

```diff
@@ -49,6 +49,7 @@
     "cynoglossus_semilaevis",
     "cyprinodon_variegatus",
     "cyprinus_carpio",
```

Comment:
in allowed_species we keep only the species/production name that we will process. Is there any reason why cyprinus_carpio is not removed or just an oversight? We need to keep the number of non strain species at 200.

Reply:
I initially left it because all the other carps have that scientific name and I wasn't sure what the purpose of this list was. I've removed it.